instrumentation/dubbo-rpc/src/main/java/brave/dubbo/rpc/TracingFilter.java

```diff
@@ -48,10 +46,14 @@
   TraceContext.Injector<Map<String, String>> injector;
 
   /**
-   * {@link ExtensionLoader} supplies the tracing implementation which must be named "tracing". For
-   * example, if using the {@link SpringExtensionFactory}, only a bean named "tracing" will be
-   * injected.
+   * <h3>Notes for com.alibaba:dubbo</h3>
+   *
+   * {@code com.alibaba.dubbo.common.extension.ExtensionLoader} supplies the tracing implementation
```

Comment:
TODO: I think maybe we can just say the word ExtensionLoader and SpringExtensionFactory here and leave out the package name as they likely exist. We should also check the README for drift